src/commands/test.js

```diff
@@ -38,6 +38,11 @@ const test = (context) => {
         commands.push(`--timeout=${global.argOpts.timeout}`);
       }
 
+      if (global.argOpts.grep) {
+        commands.push('--');
```

Comment:
If `global.argOpts.timeout` is also set, I believe this will fail, right? If so, please add a condition here, otherwise, please ignore my comment.

Reply:
Stuuuuupid mistake, fixed.